lib/analyses_to_run_in_stata.csv

```diff
@@ -0,0 +1,10 @@
+﻿outcome,cohort,subgroup,time_periods
+t2dm,pre_vaccination,hospitalised,reduced
```

Comment:
[Optional] Please could this be 'prevax' and 'vax' or whatever is consistent with the data files and R code.